src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,46 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File fromFile = new File(fromFileName);
+        File toFile = new File(toFileName);
+        String[] strings;
+        try {
+            strings = Files.readAllLines(fromFile.toPath()).toArray(new String[0]);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from the file " + fromFileName, e);
+        }
+        try {
+            toFile.createNewFile();
+            WorkWithFile workWithFile = new WorkWithFile();
+            String[] outputString = workWithFile.outputString(strings);
```

Comment:
```suggestion
            String[] outputString = outputString(strings);
```

we could call methods of this class just by their name, we don't need to create istance of this class